run_conf/bstack-local-parallel.json

```diff
@@ -0,0 +1,50 @@
+{
+  "auth": {
+    "username": "BROWSERSTACK_USERNAME",
+    "access_key": "BROWSERSTACK_ACCESS_KEY"
+  },
+  "browsers": [
+    {
+      "browser": "chrome",
+      "os": "Windows 10",
+      "versions": ["88"]
+    },
+    {
+      "browser": "firefox",
+      "os": "OS X Big Sur",
+      "versions": ["86"]
+    },
+    {
+      "browser": "edge",
+      "os": "Windows 10",
+      "versions": ["88"]
+    },
+    {
+      "browser": "chrome",
+      "os": "OS X Catalina",
+      "versions": ["89"]
```

Comment:
Lets update this with "latest" to be futureproof

Reply:
done